testing/fil_token_integration/actors/basic_nft_actor/src/lib.rs

```diff
@@ -0,0 +1,76 @@
+use fil_fungible_token::runtime::blockstore::Blockstore;
```

Comment:
Currently fil_fungible_token implements a SDK-delegated blockstore. 

- We could reimplement (copy-paste) this blockstore in the frcxx_nft library. But that brings the dependency on `anyhow` into our library code
- Or perhaps doesn't need to be in either of our libraries at all? It's a minimal interface to implement so can be done purely in actor example code
- We could even just import a library such https://github.com/polyphene/fvm-rs-sdk/blob/main/sdk/src/state/cbor.rs that implements it

Reply:
Wanna see if you can go fix FVM to remove anyhow from this? It might not be easy tho